samcli/commands/build/command.py

```diff
@@ -101,6 +107,21 @@
     help="If your functions depend on packages that have natively compiled dependencies, use this flag "
     "to build your function inside an AWS Lambda-like Docker container",
 )
+@click.option(
+    "--container-env-vars",
+    "-e",
+    default=None,
+    multiple=True,  # Can pass in multiple env vars
+    required=False,
+    help="Path to environment variable json file (ex: env_vars.json) to pass into build containers",
```

Comment:
Why this help text looks exactly the same as `--container-env-vars-file?`

Reply:
My bad, pasted the wrong text here, will ship a fix soon